indico/modules/rb_new/client/js/components/App.jsx

```diff
@@ -84,13 +84,53 @@ class App extends React.Component {
         iconName: 'home'
     };
 
+    state = {
+        userActionsVisible: false
```

Comment:
`sidebarVisible`?

Reply:
(merging anyway, since this is a minor change that's not super important, and i already rebased it)